docs/source/building.rst

```diff
@@ -225,8 +278,18 @@ Regression Manager
     cocotb will *not* apply its own default coverage collection settings,
     like enabling branch coverage and excluding files in the cocotb package directory.
 
+    .. versionadded:: 2.0
+
+.. envvar:: COVERAGE_RCFILE
+
+    See :envvar:`COCOTB_COVERAGE_RCFILE`.
+
     .. versionadded:: 1.7
 
+    .. deprecated:: 2.0
+
+        :envvar:`COVERAGE_RCFILE` is renamed to :envvar:`COCOTB_COVERAGE_RCFILE`.
```

Comment:
While this PR is documenting the current code correctly, I think the PR that did the renaming of this variable in the code has slightly broken the coverage functionality.
Unless I'm misunderstanding things, we did rely on `coverage.py` to read the original name `COVERAGE_RCFILE` (see https://coverage.readthedocs.io/en/latest/config.html#configuration-reference), which it won't do for `COCOTB_COVERAGE_RCFILE`.

Reply:
I opened #3947 for this.